market/uipath/demo/Jenkinsfile

```diff
@@ -0,0 +1,41 @@
+pipeline {
+  agent {
+    docker {
+      image 'axonivy/build-container:web-1.0'
+    }
+  }
+
+  options {
+    buildDiscarder(logRotator(numToKeepStr: '60', artifactNumToKeepStr: '10'))
+  }
+
+  triggers {
+    cron '@midnight'
+  }
+
+  parameters {
+    string(name: 'engineListUrl',
+      description: 'Engine to use for build',
+      defaultValue: 'https://jenkins.ivyteam.io/job/ivy-core_product/job/master/lastSuccessfulBuild/')
+  }
+
+  stages {
+    stage('build') {
+      steps {
+          script {
+            def mavenParameters = "-Divy.engine.list.url=${params.engineListUrl} " +
+              "-Divy.engine.version=[9.2.0,] -Dmaven.test.failure.ignore=true " +
+              "-f market/uipath/demo/pom.xml"

```

Comment:
@alexsuter  maybe you know a nicer solution to this? I'd prefer to set another work.dir for the whole build chain ... resting where the Jenkinsfile is. But didn't quickly see who to do it.

Reply:
sample build: https://jenkins.ivyteam.io/job/market-connector-uipath/job/feature%252FXIVY-5005-jenkins-build/

Reply:
you can use 
dir('my-working-dir') {
    // do what ever i like
}